src/App.vue

```diff
@@ -607,6 +619,15 @@ export default class App extends Mixins(
       // fetch the draft filing and resources
       try {
         if (this.getBusinessId) {
+          try {
+            await this.fetchBusinessData() // throws on error
+          } catch (error) {
```

Comment:
Why not let the outer catch block handle this (line 649)?

Reply:
it appears to break some of the tests if I do this

Reply:
Maybe the tests should be updated as well?

Reply:
updated the tests

Reply:
Awesome, thanks!